windows/wrapper/impl_webrtc_VideoCapturer.cpp

```diff
@@ -969,7 +1077,8 @@ namespace webrtc
     display_orientation_(nullptr),
     video_encoding_properties_(nullptr),
     media_encoding_profile_(nullptr),
-    subscriptions_(decltype(subscriptions_)::create())
+    subscriptions_(decltype(subscriptions_)::create()),
+    pool_(new I420BufferPool())
```

Comment:
>new  [](start = 10, length = 4)

std::make_unique

Reply:
Yes, I guess it makes it a bit more explicit and less error-prone.